gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Mon Aug 31 15:52:25 CEST 2020
-distributionBase=GRADLE_USER_HOME
-distributionPath=wrapper/dists
-zipStoreBase=GRADLE_USER_HOME
-zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.5-bin.zip
+#Sun Mar 14 11:30:07 CET 2021
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists

```

Comment:
what is ^M

Reply:
I was doing that on my Windows machine and is a `\r\n` line ending.